packages/plugins/ansible-runner/ansible-runner.spec

```diff
@@ -26,6 +26,7 @@ standalone tool, or imported into a python project.
 Summary:        %{summary}
 %{?python_provide:%python_provide python%{python3_pkgversion}-%{pypi_name}}
 Obsoletes:      python38-%{pypi_name} < %{version}
+Obsoletes:      python3-%{pypi_name} < %{version}
```

Comment:
IMHO the obsoletes now should be on the `ansible-runner` package, as that is the one providing the binaries (and thus the same functionality) now.